x/upgrade/keeper/keeper.go

```diff
@@ -28,6 +29,7 @@ type Keeper struct {
 	storeKey           sdk.StoreKey
 	cdc                codec.BinaryMarshaler
 	upgradeHandlers    map[string]types.UpgradeHandler
+	ModuleManager      *module.Manager
```

Comment:
Also, the type here should be a `moduler.VersionManager` interface (i.e. an interface with a single `GetConsensusVersions` method).

Reply:
fixed in new commit - new interface added as per spec